templates/deprecated-windows.tpl.yaml

```diff
@@ -62,6 +62,9 @@ metadata:
   labels:
     template.kubevirt.io/type: "base"
     template.kubevirt.io/version: "{{ lookup('env', 'VERSION') | default('devel', true) }}"
+{% if item.default %}
```

Comment:
I would not add this label to deprecate templates

Reply:
I have a PR open to remove those templates, shouldn't be a problem for 0.13.0

Reply:
https://github.com/kubevirt/common-templates/pull/275